audio/descriptors.c

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright (C) 2017 Kyle Robbertze
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "descriptors.h"
+#include "date.h"
+
+__code __at(DSCR_AREA) struct usb_descriptors code_descriptors = {
+    .device = {
+        .bLength            = USB_DT_DEVICE_SIZE,
+        .bDescriptorType    = USB_DT_DEVICE,
+        .bcdUSB             = USB_BCD_V20,
+        .bDeviceClass       = 0,
+        .bDeviceSubClass    = 0,
+        .bDeviceProtocol    = 0,
```

Comment:
The reference values are for a composite device (see [TODO](https://github.com/timvideos/HDMI2USB-fx2-firmware/pull/23/files#diff-fe51caa755e18b3c02004e8eb9f65b7bR1) for explanation), while the audio spec requires these to be 0 for audio-only devices, which this currently is